pages/docs/tutorials/kafka/bindings-with-kafka.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Kafka-specific Bindings
+description: In this tutorial, you'll learn how how to add specific Kafka information to an AsyncAPI document using bindings.
+---
+
+## Introduction
+
+In a previous tutorial, you learnt how to manage schemas with schema registry. In this tutorial, you will learn how bindings with Kafka works by defining Kafka message key content and extending your AsyncAPI document with protocol-specific information.
+
+## Background Context
+
```

Comment:
For the background context...
- Define the general concept of `bindings`
- Tell me it's relationship to the AsyncAPI document
- Define Kafka `bindings`